ftplugin/ruby.vim

```diff
@@ -111,7 +111,7 @@ else
   if !exists('g:ruby_default_path')
     if has("ruby") && has("win32")
       ruby ::VIM::command( 'let g:ruby_default_path = split("%s",",")' % $:.join(%q{,}) )
-    elseif executable('ruby')
+    elseif executable('ruby') && exists('$HOME')
```

Comment:
Might also want to use `!empty()`, since an empty `$HOME` also results in the same error then (`E344: Can't find directory "HOME" in cdpath`).